warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -12,18 +12,15 @@ models:
             - date
 
   - name: int_gtfs_quality__no_rt_critical_validation_errors
-    description: |
-      Checks for any critical validation notices in RT feeds.
-    tests:
+    tests: &stg_gtfs_guideline_tests_rt
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - date
             - base64_url
-    columns:
-      - name: status
-        tests:
-          - dbt_utils.not_null_proportion:
-              at_least: 0.99
+            - feed_type
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('int_gtfs_quality__schedule_feed_guideline_index')
```

Comment:
I don't think that this is necessarily expected to have an equal row count... We expect to have a moderately different number of schedule and RT feeds, I think?